synapse/api/auth.py

```diff
@@ -810,6 +810,10 @@ def validate_macaroon(self, macaroon, type_string, verify_expiry, user_id):
         else:
             v.satisfy_general(lambda c: c.startswith("time < "))
```

Comment:
and put a comment explaining why we aren't ever going to check the ``"time < "`` caveats.